src/class_addressGenerator.py

```diff
@@ -27,8 +27,9 @@ class addressGenerator(StoppableThread):
     def stopThread(self):
         try:
             queues.addressGeneratorQueue.put(("stopThread", "data"))
-        except:
-            pass
+        except queues.Full:
```

Comment:
no member

Reply:
(I'm guessing, verify please)
```
from six import queue.Full
```
and then here
```
except Full:
```